From cda86e34ddadba5f08dea764881bbb6c8ec01e8a Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sat, 25 Jan 2020 02:00:43 +0000
Subject: [PATCH] Fix building with system zlib

Add zlib_config target because perfetto depends on it.

Bug: 977964
Change-Id: I2aac5c3704f1274f932a61264d1c26a05f2db55e
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2019035
Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
Commit-Queue: Thomas Anderson <thomasanderson@chromium.org>
Cr-Commit-Position: refs/heads/master@{#735201}
---
 build/linux/unbundle/zlib.gn | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build/linux/unbundle/zlib.gn b/build/linux/unbundle/zlib.gn
index 1fdd472f35e..97d6aebe6d5 100644
--- a/build/linux/unbundle/zlib.gn
+++ b/build/linux/unbundle/zlib.gn
@@ -13,6 +13,10 @@ config("system_zlib") {
   defines = [ "USE_SYSTEM_ZLIB=1" ]
 }
 
+config("zlib_config") {
+  configs = [ ":system_zlib" ]
+}
+
 source_set("zlib") {
   deps = [ ":zlib_shim" ]
   libs = [ "z" ]
